### ISSUE TITLE
Context Sensitive Help in NVDA Dialogs

### ISSUE THREAD

**Issue by ThomasStivers** [nvaccess/nvda#7757]
### Steps to reproduce:

* NVDA has a well thought out user guide, but it is prohibitively long for new users. A new user needs to be able to get help relevant to what they are doing while they are working in various dialogs and controls with NVDA.
* Some controls in NVDA preferences dialogs have long labels which cause them not to fit well visually (see #6345). These labels can be shortened if additional help for these controls is readily available with F1 or by pressing a Help button.

### Expected behavior:

1. Explicit anchors should be added to all sections of the NVDA User Guide so that it can be opened to the correct place when context help is invoked.
2. When in an NVDA preference dialog a Help button should be available to open the NVDA User Guide  for all controls in that dialog.
3. Optionally, pressing F1 on any control within an NVDA Preference dialog should open the NVDA User Guide to the section describing the function of that specific control if one exists, or for the dialog if no explicit help exists.

### Actual behavior:

Currently NVDA's preferences dialogs do not have a help button and do not respond to the F1 key. The NVDA User Guide explains NVDA's dialogs and controls in detail, but new users find it daunting to locate information quickly.

### Similar issues which this is not:

* Providing context sensitive help for NVDA's dialogs is not a tutor mode as in #3232.
* This issue is not intended to provide general Windows help as in #2699.

**Comment by feerrenrut** [comment]
This sounds like a good idea. Happy to accept a PR for this, though I would suggest the work is split into 3 PR's one for each of the points listed under expected behaviour.

**Comment by leonardder** [comment]
If we want to use context sensitive help from the user guide, I belief it would help a lot if we'd show the user guide in a ui.browseableMessage

**Comment by PratikP1** [comment]
@ThomasStivers commented:
> 3.	Optionally, pressing F1 on any control within an NVDA Preference dialog should open the NVDA User Guide to the section describing the function of that specific control if one exists, or for the dialog if no explicit help exists.

It seems like pulling up the entire user guide and pointing to specific sections for each control might be a more intensive process. I have thought for some time that NVDA help in the form of a long document, while extremely useful, is becoming less manageable from a usability perspective. I would encourage NVAccess to consider a modular approach to help and documentation that could support tutorial work as well as the current set of user documentation.

**Comment by josephsl** [comment]
Hi, overall, this is similar in scope to (now discontinued) Control Usage Assistant add-on with help on NVDA options added. CC @DerekRiemer
